commands/info.md

```diff
@@ -315,6 +318,8 @@ Here is the meaning of all fields in the **clients** section:
 *   `total_watched_keys`: Number of watched keys. Added in Valkey 8.0.
 *   `total_blocking_keys`: Number of blocking keys.
 *   `total_blocking_keys_on_nokey`: Number of blocking keys that one or more clients that would like to be unblocked when the key is deleted.
+*   `paused_reason`: The current paused reason of the instance: "client_pause" means trigger by `CLIENT PAUSE`,
+    "shutdown_in_progress", "failover_in_progress" and "none" means no clients are paused. Added in Valkey 8.1.
```

Comment:
please help and insert the suggestion on this sentence :)

Reply:
I think your text is acceptable. Not perfect maybe but we have many things that are not perfect...